utils/fields.py

```diff
@@ -262,10 +262,23 @@ def generate_fake_data(self, faker, overrides=None, require_all=False):
         if self.discriminator and d_value:
             overrides[self.discriminator] = d_value
 
+        # Address fields are related so we want to have city, state, zip values that are valid instead of independent
+        # random ones.  This relies on the swagger definition using the name "Address" and the properties being named
+        # the same across APIs.
+        virtual_city_state_zip = None
+        if self.is_address_object():
+            virtual_city_state_zip = BaseAPIField(
```

Comment:
My solution here was to not subclass the object field just to handle the address definition because it would have been a lot of redundancy and boilerplate to copy and it didn't seem to fit with how we are creating our class types.

I created a virtual object field `city_state_zip` that will return the 3 valid values at once instead of how each used to get called individually and not match one another.